README.md

```diff
@@ -43,7 +45,80 @@ Run the app with: `react-native run-ios` from the root of the project.
 
 ## Available methods
 
-`initialize(string endpoint, string method (post or get), string protocol (https or http), string namespace, string appId)`
+Note:
+
+Optional string arguments if not set must be specified as `null`.
+Optional object arguments must be provided as an empty object if none are set.
+Optional keys within an object argument can be omitted if not set.
+Optional array arguments must be provided as empty arrays if not set.
```

Comment:
Looking at [how it looks](https://github.com/snowplow-incubator/snowplow-react-native-tracker/blob/release/0.1.0-alpha.13/README.md#available-methods) , I don't think it is what you want. I think we need additional new line after each sentence so that they will appear as they do here.

Reply:
Nice catch, thanks, will sort this!